test/units/galaxy/test_collection_install.py

```diff
@@ -297,6 +297,26 @@ def test_build_requirement_from_tar(collection_artifact):
     assert actual.src == to_text(collection_artifact[1])
     assert actual.ver == u'0.1.0'
 
+def test_build_requirement_from_tar_url(tmp_path_factory):
+    test_dir = to_bytes(tmp_path_factory.mktemp('test-ÅÑŚÌβŁÈ Collections Input'))
+    concrete_artifact_cm = collection.concrete_artifact_manager.ConcreteArtifactsManager(test_dir, validate_certs=False)
+    test_url = 'https://github.com/org/repo/sample.tar.gz'
```

Comment:
Maybe, instead of GitHub, use `example.com` or something that is guaranteed not to start having a file in the future?